lib/animina_web/live/profile_live.ex

```diff
@@ -120,25 +120,29 @@ defmodule AniminaWeb.ProfileLive do
       <div :if={@user != nil}>
         <div class="flex items-center px-4 space-x-4 border border-gray-100 rounded-lg shadow-sm">
           <div class="py-4">
-            <h3 class="text-lg font-semibold"><%= @user.name %></h3>
-            <p class="text-sm font-medium text-gray-500">@<%= @user.username %></p>
+
+            <h3 class="text-lg dark:text-white font-semibold"><%= @user.name %></h3>
+            <p class="text-sm dark:text-gray-100 font-medium text-gray-500"><%= @user.username %></p>
 
             <div class="mt-2">
-              <p class="text-gray-600">
-                <%= gettext("Lives in ") %> <%= @user.city.zip_code %> <%= @user.city.name %>.
+              <p class=" dark:text-gray-100  text-gray-600">
+                <%= gettext("Lives in ") %> <%= @user.city.name %>
+              </p>
+               <%= if @user.occupation && @user.occupation != "" do %>
+
+              <p class=" dark:text-gray-100  text-gray-600">
+                <%= gettext("I'm a ") %> <%= @user.occupation %>
+                
               </p>
+              <%end %>
 
-              <%= if @user.occupation && @user.occupation != "" do %>
```

Comment:
No need to replace this code. I don't want to display the occupation part if it is nil or empty.

Reply:
I have made this change